R/descriptives.R

```diff
@@ -1632,14 +1632,25 @@ DescriptivesInternal <- function(jaspResults, dataset, options) {
   ))
 }
 
+# Formula for chi-square confidence intervals for std. dev. and variance found here: https://www.itl.nist.gov/div898/handbook/prc/section2/prc231.htm
```

Comment:
This link gives a 503 service unavailable for me. Perhaps https://en.wikipedia.org/wiki/Standard_deviation#Confidence_interval_of_a_sampled_standard_deviation and https://en.wikipedia.org/wiki/Variance#Distribution_of_the_sample_variance are a better reference?